dags/public-health/covid19/get_help_to_esri.py

```diff
@@ -0,0 +1,236 @@
+        The start date of the history (defaults to April 8, 2020)
+    end_date: datetme.date
+        The end date of the history (defaults to the present day)
+
+    Returns
+    =======
+    A dataframe with the history for the given facility.
+    """
+    TOKEN = Variable.get("GETHELP_OAUTH_PASSWORD")
+    start_date = start_date or datetime.date(2020, 4, 8)
+    end_date = end_date or pandas.Timestamp.now(tz="US/Pacific").date()
+
+    # Get the shelter bed program ID
+    res = make_get_help_request(f"facilities/{facility_id}/facility-programs", TOKEN)
+    programs = pandas.io.json.json_normalize(res)
+    assert len(programs)
+    shelter_programs = programs[programs.name.str.lower().str.contains("shelter bed")]
+    assert len(shelter_programs) == 1
+    program_id = shelter_programs.iloc[0]["id"]
+
+    # Get the history stats for the shelter bed program
+    res = make_get_help_request(
+        f"facilities/{facility_id}/facility-programs/{program_id}/statistics",
+        TOKEN,
+        params={"startDate": str(start_date), "endDate": str(end_date)},
+    )
+    history = pandas.io.json.json_normalize(res)
+
+    # Add ID column so we can filter by them later
+    history = history.assign(program_id=program_id,)
+    return history
+
+
+def assemble_facility_history(facility):
+    """
+    Given a facility, assemble its history stats into a dataframe.
+    This is the same as get_facility_history, but also adds some
+    additional columns from the facility data.
+
+    Parameters
+    ==========
+    facility: pandas.Series
+        A row from the facilities dataframe
+
+    Returns
+    =======
+    A dataframe with facility history.
+    """
+    history = get_facility_history(facility["id"])
+    if not len(history):
+        return None
+    history = history.assign(
+        facility_id=facility["id"],
+        name=facility["name"],
+        phone=facility["phone"],
+        website=facility["website"],
+        address=facility["address1"],
+        city=facility["city"],
+        county=facility["county"],
+        state=facility["state"],
+        zipCode=facility["zipCode"],
+        latitude=facility["latitude"],
+        longitude=facility["longitude"],
+    ).drop(columns=["id"])
+    return history
+
+
+def assemble_get_help_timeseries():
+    """
+    Gets a full timeseries for all facilities managed by the GetHelp system.
+    """
+    df = pandas.DataFrame()
+    facilities = get_facilities()
+    for idx, facility in facilities.iterrows():
+        history = assemble_facility_history(facility)
+        if history is not None:
+            df = df.append(history)
+    df = df.assign(
+        dataDate=pandas.to_datetime(df.dataDate)
+        .dt.tz_localize("US/Pacific")
+        .dt.tz_convert("UTC")
+    ).sort_values(["facility_id", "dataDate"])
+    return df
+
+
+def load_get_help_data(**kwargs):
+    facilities = get_facilities()
+    upload_to_esri(facilities, FACILITIES_ID, "/tmp/facilities.csv")
```

Comment:
I think this should probably be only open facilities, rather than all that might be in teh system./

Reply:
My intention here was that we could easily filter by `status != 0` in the AGOL dashboard.

Reply:
easy enough. Let's add stuff to the `README` include status codes, get help spreadsheet, etc

Reply:
Makes sense.

Reply:
added some docs.